tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -355,6 +355,34 @@ public function getFolderId(string $user, string $spaceName, string $folderName)
 		return $responseArray[0][0];
 	}
 
+	/**
+	 * @param string $user
+	 * @param string $spaceName
+	 *
+	 * @return mixed
+	 * @throws GuzzleException
+	 * @throws JsonException
+	 */
+	public function getPrivateLink(string $user, string $spaceName): string {
+		$this->setSpaceIDByName($user, $spaceName);
+		$this->featureContext->setResponse(
```

Comment:
Is it really necessary to set the response here? Also, helper functions shouldn't set the response instead the response should be set in the When step functions. This will prevent future side effects and refactoring.
```suggestion
```